tests/test_preproc.py

```diff
@@ -1721,7 +1722,21 @@ def test_sync_ts_data_timestamps(self):
         dt = sync_timestamp[1] - sync_timestamp[0]
         num_padding = np.arange(sync_timestamp[0]-dt,0,-dt).shape[0]
         self.assertTrue(sync_data.shape[0], num_padding + sync_timestamp.shape[0])  
-              
+    
+    def test_destripe_lfp_batch(self):
+        save_path = Path(data_dir) / 'test_a'
+
+        data = np.random.randint(0,50, (10000,384))
+        sample_rate = 1000
+        bit_volts=0.1
+
+        destripe_lfp_batch(data, save_path, sample_rate, bit_volts, batch_length = 180, dtype='int16')
+        destriped_data = np.memmap(save_path, shape=data.shape, dtype='int16')
+
+        print(np.all(data.shape == destriped_data.shape))
+        del destriped_data # delete memmap file to completely delete the file later
```

Comment:
Tests should use an 'assert' so that the test fails if the condition isn't true.